target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -80,6 +80,10 @@ xzwifi,creativebox-v1)
 xiaomi,redmi-router-ac2100)
 	ucidef_set_led_netdev "wan" "wan" "$boardname:white:wan" "wan"
 	;;
+xiaomi,mi-router-ac2100)
+	ucidef_set_led_switch "wan-yellow"  "WAN (yellow)"  "$boardname:yellow:wan"  "switch0" "0x01" "0x01"
+	ucidef_set_led_switch "wan-blue"  "WAN (blue)"  "$boardname:blue:wan"  "switch0" "0x01"
```

Comment:
Can you explain what you do here? (Despite, please replace the double spaces by single ones.)

Reply:
Inteded usage of the speed masks at the WAN LED's definition:
If a cable is connected but the speed can't be recognized the LED will be yellow. If a cable is connected and the speed can be recognized (10mbps, 100mbps, 1000mbps) the light will be blue. By this you can easily get a hint on cabling problems.